CHANGELOG.md

```diff
@@ -484,3 +488,4 @@
 [@theunraveler]: https://github.com/theunraveler
 [@pirj]: https://github.com/pirj
 [@vitormd]: https://github.com/vitormd
+[@mattmccormick]: https://github.com/mattmccormick
```

Comment:
Can you remove it? Because it is not necessary for the above new changelog entry format style :-)

Reply:
Removed